features/child_workflow/signal/feature.java

```diff
@@ -13,37 +11,36 @@
 import io.temporal.workflow.Workflow;
 import io.temporal.workflow.WorkflowInterface;
 import io.temporal.workflow.WorkflowMethod;
-import java.time.Duration;
 
 @WorkflowInterface
 public interface feature extends Feature {
 
   @WorkflowInterface
-  public interface ChildWorkflow {
+  interface ChildWorkflow {
 
     @WorkflowMethod
     String workflow();
 
     @SignalMethod
     void unblock(String message);
-  }
 
-  class ChildWorkflowImpl implements ChildWorkflow {
-    /*
-     * A workflow that waits for a signal and returns the data received.
-     */
+    class Impl implements ChildWorkflow {
+      /*
+       * A workflow that waits for a signal and returns the data received.
+       */
 
-    private String childWorkflowUnblockMessage;
+      private String childWorkflowUnblockMessage;
 
-    @Override
-    public String workflow() {
-      Workflow.await(() -> childWorkflowUnblockMessage != null);
-      return childWorkflowUnblockMessage;
-    }
+      @Override
+      public String workflow() {
+        Workflow.await(() -> childWorkflowUnblockMessage != null);
+        return childWorkflowUnblockMessage;
+      }
 
-    @Override
-    public void unblock(String message) {
-      childWorkflowUnblockMessage = message;
+      @Override
+      public void unblock(String message) {
+        childWorkflowUnblockMessage = message;
+      }
```

Comment:
The diff here is mostly indentation, moving an impl into the interface - easier to understand with hide-whitespace on.